libs/drawer/src/index.tsx

```diff
@@ -47,7 +47,31 @@ const Drawer = ({
           testId="drawerCloseButton"
           top="8px"
         />
-        <Stack direction="column" height="100%" spacing={4} sx={{ '&>*': { bg: drawerBackground } }}>
+        <Stack
+          direction="column"
+          height="100%"
+          overflow="auto"
+          spacing={4}
+          sx={{
+            '::-webkit-scrollbar': {
```

Comment:
Do we need `-webkit` here? I thought Chakara-UI should handle that for different browsers.

Reply:
My understanding is scrollbar styling is non-standard/experimental; I don't believe Chakra handles this or is even able to. `::webkit-scrollbar` is [not supported on Firefox](https://caniuse.com/?search=%3A%3A-webkit-scrollbar). `scrollbar-*` is also [experimental](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars) with low browser support, it also does not provide granular styles. As mentioned, this is a temporary solution in lieu of responsive styles.

Reply:
Just like what we agreed in dev&designer sync meeting, we definitely need to replace the drawer with other UI such as pop over or modal. So we need another issue for it.

Reply:
https://github.com/dolbyio-samples/rts-app-react-publisher-viewer/issues/242